### DISCUSSION TITLE
Hombli Radiator Thermostat

### DISCUSSION THREAD

**Discussion by dimitribo** [make-all/tuya-local#1861] [Q&A]
Hello,

I added the Hombli Radiator Thermostats with this integration. Finally, afther trying almost now 1 year to integrate them with other integrations (local tuya, tuya BLE, python, ...) there is now a integration how does the job! Thanks !!!

I have 10 thermostats. 
At this moment, for testing, i have 3 thermostats connected true a USB hub and 3 thermostats are connected true a Bridge. (The maximum true a hub or bridge is 3, when i add a 4th thermostat it doesn't work...) 
I can receive de data and also change temperature or settings whitch is good.
But, at this moment, all the values are only updated when i change something on the thermostat (a setting, the target temperature, ...) 
So when i look in the trendings, i get all the time the same measured and target values.
I guess its something to save on the battery consumption ... but is it possible to add a time to refresh the values ?
Somebody nows ho to solve this please ?

Kind regards !

**Comment by make-all** [comment]
The per-hub connection limit and maybe the other issue with updates may be improved (but no guarantee of an absolute fix) by selecting the "Poll only" option in the device configs so the connections are not being held open.
